### DISCUSSION TITLE
RobTop dev mode GDPS Geode mod (Cross compatible with iOS, Mac, Windows, and Android)

### DISCUSSION THREAD

**Discussion by ghost** [geode-sdk/sdk#575] [Mod Ideas]
A mod where you have the RobTop buttons instead of the moderator buttons for a GDPS only
You have the second star button the green star above the regular which is shown what it does in Rob's streams
Then you have Dev: suggest stars where you can rate a level
Not sure if this is necessary but maybe a second play button next to the original one for online levels to noclip a level
check robtop's streams for how his developer tools work there are many saved in youtube

**Comment by heavyutils** [comment]
Cool this should exist. Even for non-gdps, to set a custom rating. And demons too.

**Comment by OmgRod** [comment]
That code is inclined unfortunately I think so no way to access it. But with how most gdps cores work I don't think this would do much
